packages/botonic-react/src/components/multichannel/multichannel.jsx

```diff
@@ -61,7 +61,11 @@ export const Multichannel = props => {
     newChildren = newChildren.map((c, index) =>
       index > 0 && typeof c === 'string' ? props.messageSeparator + c : c
     )
-    newChildren = <Text key={props.key}>{newChildren}</Text>
+    newChildren = (
+      <Text key={props.key} {...MULTICHANNEL_WHATSAPP_PROPS}>
```

Comment:
Here, if we put this, it will always have markdown=false, right?? Then if for example you have a bot, working in webchat, the markdown will not be applied?

Reply:
For "real webchat", we won't reach here due to previous guard:
```
  if (!isWhatsapp(requestContext)) {
    return props.children
  }

```
When debugging "as whatsapp" from dev machine, we will reach here and the flag will disable the transformation from markdown syntax to html.